quickstart/dotnet-core/Program.cs

```diff
@@ -49,7 +49,7 @@ class Program
 
         public static int RetryMaxAttempts => 5;
         
-        public static ConnectionMultiplexer Connection { get { return _connection; } }
+        public static ConnectionMultiplexer Connection { get { return _connection ?? CreateConnectionAsync().GetAwaiter().GetResult(); } }
 
```

Comment:
can simplify to:
        public static ConnectionMultiplexer Connection => _connection ?? CreateConnectionAsync().GetAwaiter().GetResult();